src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -735,7 +735,7 @@ private async Task RunSubscriptionAsync()
                                 var nextNodeIndex = (_forcedTopologyUpdateAttempts++) % curTopology.Count;
                                 try
                                 {
-                                    (_, _redirectNode) = await reqEx.GetRequestedNode(curTopology[nextNodeIndex].ClusterTag).ConfigureAwait(false);
+                                    (_, _redirectNode) = await reqEx.GetRequestedNode(curTopology[nextNodeIndex].ClusterTag, true).ConfigureAwait(false);
```

Comment:
please name the `bool` parameter like
`Foo( ... , throwOnError: true)`

Reply:
Done